charts/admission-controller/values.yaml

```diff
@@ -101,6 +103,8 @@ scanner:
   # Put your <Sysdig-Secure-Secret-Token> in this value
   # Set to http://proxyIp:proxyPort in case connection to Sysdig Secure requires a proxy
   httpProxy: ""
+  # Set to https://proxyIp:proxyPort in case connection to Sysdig Secure requires a proxy
```

Comment:
Better use plain http:// in the example...

Reply:
Updated both the example and the version in the Chart. We're on the road again. Thanks @airadier!